RecoLocalTracker/SiPixelClusterizer/interface/SiPixelFedCablingMapGPU.h

```diff
@@ -8,20 +8,19 @@ namespace pixelgpudetails {
   constexpr unsigned int MAX_LINK =  48;  // maximum links/channels for Phase 1
   constexpr unsigned int MAX_ROC  =   8;
   constexpr unsigned int MAX_SIZE = MAX_FED * MAX_LINK * MAX_ROC;
-  constexpr unsigned int MAX_SIZE_BYTE_INT  = MAX_SIZE * sizeof(unsigned int);
   constexpr unsigned int MAX_SIZE_BYTE_BOOL = MAX_SIZE * sizeof(unsigned char);
 }
 
 // TODO: since this has more information than just cabling map, maybe we should invent a better name?
-struct SiPixelFedCablingMapGPU {
+struct alignas(128) SiPixelFedCablingMapGPU {
```

Comment:
This applies the alignment to the structure as a whole, but not to the individual data members, so they won't be aligned to 128-bit boundary if `MAX_SIZE` is not a multiple of 16.

Reply:
@makortel ?

Reply:
Right, I had misunderstood. I'll make a fix according to your suggestion shortly (even if in this case the `alignas(128)` has little effect as `pixelgpudetails::MAX_SIZE % 128 == 0`).

Reply:
Done, and rebased on top of 10_6_X_Patatrack.